### DISCUSSION TITLE
Turborepo Seal Doesn't Appear

### DISCUSSION THREAD

**Discussion by kaganergovan** [vercel/community#7176] [Help]
### Summary

Hello friends!

On my current Vercel builds, Turborepo seal doesn't appear while building. Is it expected behavior or I made some configuration error?

### Steps to Reproduce

I didn't do much to reproduce, just sent new commit to re-deploy build.

**Comment by versecafe** [comment]
Are you referring to on the Build Logs, the little summary it should show

<img width="1412" alt="image" src="https://github.com/vercel/community/assets/147033096/385c5bf3-34d9-418b-9995-457ae83d6000">

If you're referring to the project card itself then that's normal it'll usually show either the framework logo or favicon of the site instead, but can sometimes default to the turbo logo if a framework is not selected and a favicon isn't found

**Reply by kaganergovan** [reply]
Ah, yes, I'm referring build logs section. However it no longer appears...

'Run Summary' with Turbo seal.

**Comment by amyegan** [comment]
Hey @kaganergovan. The "Run Summary" appears on each of the test repos I tried. Do you have an example repo I can use to reproduce this issue?

**Reply by kaganergovan** [reply]
Hi @amyegan! Could you please try my previous [test repo](https://github.com/kaganergovan/test-ex)

Thanks!

[a comment by an automated account is not shown]
